### DISCUSSION TITLE
Feature Request: Task Duration

### DISCUSSION THREAD

**Discussion by CodeCity** [thesamim/TickTickSync#123] [Ideas]
In TickTick, I use the calendar so I can time block. This requires a duration for most tasks. 

For example, I start off each day with a 30 minute block to check my email, reply, and create tasks. I have a one hour block in TickTick for my lunch. Some projects get 4-5 hour blocks assigned where I am focused on one thing. 

Unfortunately, TickTickSync assumes all of my tasks are 15 minutes, and it has removed the task duration on my tasks. (I'm not upset, I knew there were risks) Having a duration on the task would be helpful. 

Looking at the Obsidian Tasks, I'm not sure how this would work. There is no option to list a time. If it is not possible, it would be best to update the ReadMe so others are aware. Hopefully it is possible!

**Comment by thesamim** [comment]
@CodeCity : FYI I'm working on this. Slowly. But I'm working on it.
